src/app/workbench/WorkbenchViews.tsx

```diff
@@ -22,127 +41,66 @@ export function WorkbenchViews({ index, type }: IWorkbenchViewsProps) {
   const { views, selection, commentsOpen, itemIDType } = ordino.workbenches[index];
   const [setRef] = useCommentPanel({ selection, itemIDType, commentsOpen, isFocused: type === EWorkbenchType.FOCUS });
 
-  let wb = null;
+  const [mosaicState, setMosaicState] = useState<any>(null);
+  const [mosaicViewCount, setMosaicViewCount] = useState<number>(1);
+
+  const [mosaicDrag, setMosaicDrag] = useState<boolean>(false);
+
+  React.useEffect(() => {
+    if (mosaicState === null) {
+      setMosaicState(views[0].uniqueId);
+      return;
+    }
+    if (views.length > mosaicViewCount) {
```

Comment:
```suggestion
    // add new view to the workbench
    if (views.length > mosaicViewCount) {
```
I suggest adding a comment to make the following condition easier to read.

Reply:
Done